pycbc/workflow/coincidence.py

```diff
@@ -106,6 +106,33 @@ def create_node(self, trig_files, bank_file, stat_files, veto_file,
         node.new_output_file_opt(seg, '.hdf', '--output-file', tags=tags)
         return node
 
+class PyCBCFindMultiifoCoincExecutable(Executable):
+    """Find coinc triggers using a folded interval method"""
+    current_retention_level = Executable.ALL_TRIGGERS
+    file_input_options = ['--statistic-files']
+    def create_node(self, trig_files, bank_file, stat_files, veto_file,
+                    veto_name, template_str, pivot_ifo, fixed_ifo, tags=None):
+        if tags is None:
+            tags = []
+        segs = trig_files.get_times_covered_by_files()
+        seg = segments.segment(segs[0][0], segs[-1][1])
+        node = Node(self)
+        node.set_memory(10000)
+        node.add_input_opt('--template-bank', bank_file)
+        node.add_input_list_opt('--trigger-files', trig_files)
+        if len(stat_files) > 0:
+            node.add_input_list_opt('--statistic-files', stat_files)
+        if veto_file is not None:
+            node.add_input_opt('--veto-files', veto_file)
+            node.add_opt('--segment-name', veto_name)
+        if pivot_ifo is not None:
```

Comment:
This isn't optional so no need for the if statements here (and the similar places)